README.md

```diff
@@ -1,6 +1,34 @@
 # notation-akv
 
-Azure Provider for Notation CLI
+Azure Provider for the Notary v2 [Notation CLI](https://github.com/notaryproject/notation)
+
+## Getting Started:
+The following summarizes the steps to configure the Azure Key Vault notation plugin, configure gatekeeper, sign and verify a container image to Azure Kubernetes Service
+
+```bash
+# Configure notation with the Azure Key Vault plugin
+notation plugin add azure-kv ~/.config/notation/plugins/azure-kv/notation-azure-kv
+
+# Add signing and verification keys to the notation configuration policy
+notation key add --name $KEY_NAME --plugin azure-kv --id $KEY_ID --kms
+notation cert add --name $KEY_NAME --plugin azure-kv --id $KEY_ID --kms
```

Comment:
Should we add a note that this is only required for remote verification?

Reply:
Once we get the key vault mounts documented into ratify, I'll change this reference. 
If the user wants to call `notation verify` while iterating, this would either be needed, or we'd need to download the public cert to a local config. Just keeping this easy for now, but yes. We'll update for local verification.